docs/iris/src/whatsnew/1.10.rst

```diff
@@ -0,0 +1,170 @@
+    any problems you uncover, such as files that will no longer load with the
+    new implementation.
+
+* :meth:`iris.experimental.regrid.PointInCell.regridder` now works across coordinate systems, including non latlon systems. Additionally, the requirement that the source data X and Y coordinates be 2D has been removed. NB: some aspects of this change are backwards incompatible.
+* Plotting non-Gregorian calendars is now supported. This adds `nc_time_axis <https://github.com/SciTools/nc-time-axis>`_ as a dependency.
+* Promoting a scalar coordinate to a dimension coordinate with :func:`iris.util.new_axis` no longer loads deferred data.
+* The parsing functionality for Cell Methods from netCDF files is available as part of the :mod:`iris.fileformats.netcdf` module.
+* Support for the NameIII Version 2 file format has been added.
+* The Mercator projection is now supported with defaults for some parameters (_false easting\_, _false northing\_ and _scale factor at projection origin\_).
+
+  * NetCDF files which define a Mercator projection where the _false easting\_, _false northing\_ and _scale factor at projection origin\_ match the defaults will have the projection loaded correctly. Otherwise, a warning will be issued for each parameter that does not match the default and the projection will not be loaded.
+  * NetCDF files which define a steroegraphic projection where the _scale factor at projection origin\_ is equal to 1.0 will have the projection loaded correctly. Otherwise, a warning will be issued and the projection will not be loaded.
+
+* The :mod:`iris.plot` routines :func:`~iris.plot.contour`, :func:`~iris.plot.contourf`, :func:`~iris.plot.outline`, :func:`~iris.plot.pcolor`, :func:`~iris.plot.pcolormesh` and :func:`~iris.plot.points` now support plotting cubes with anonymous dimensions by specifying the *numeric index* of the anonymous dimension within the ``coords`` keyword argument.
+
+  Note that the axis of the anonymous dimension will be plotted in index space.
+* NetCDF loading and saving now supports Cubes that use the LambertConformal coordinate system.
+* The experimental structured Fieldsfile loader :func:`~iris.experimental.fieldsfile.load` has been extended to also load structured PP files.
+
+  Structured loading is a streamlined operation, offering the benefit of a significantly faster loading alternative to the more generic :func:`iris.load` mechanism.
+
+  Note that structured loading is not an optimised wholesale replacement of :func:`iris.load`. Structured loading is restricted to input containing contiguously ordered fields for each phenomenon that repeat regularly over the same vertical levels and times. For further details, see :func:`~iris.experimental.fieldsfile.load` 
+* :mod:`iris.experimental.regrid_conservative` is now compatible with ESMPy v7.
+* Zonal (i.e. longitudinal) mean support has been added to the LBPROC save rule for pp files.
```

Comment:
I think this could be better expressed
Not your fault, I know!
How about "Saving zonal (i.e. longitudinal) means to PP files now sets the '64s' bit in LBPROC."